mpiPi.h.in

```diff
@@ -340,14 +358,14 @@ extern void *saved_ret_addr;
 /*
  * Unlike some of the other platforms, we do not use setjmp to obtain
  * the frame pointer of the current function.  Instead, we use the
- * Cray intrinsic function _read_fp() to get the frame pointer of the 
```

Comment:
Not sure how about maintainers, but I'd suggest removing spacing changes from the PR to highlight its semantics.